daffodil-cli/src/it/scala/org/apache/daffodil/parsing/TestCLIParsing.scala

```diff
@@ -695,4 +695,13 @@ class TestCLIparsing {
     } (ExitCode.Success)
   }
 
+  @Test def test_Invalid_Configuration_File(): Unit = {
+    val schema = path("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/single.xsd")
+    val config = path("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/single_conf_bad.txt")
+
+    runCLI(args"parse -s $schema -c $config") { cli =>
+      cli.sendLine("0", inputDone = true)
+      cli.expectErr("Unable to load configuration")
+    }(ExitCode.ConfigError)
```

Comment:
Rest of tests have a whitespace character separating } and (, please add one here.

Reply:
Updated